tests/test_api.py

```diff
@@ -181,27 +181,53 @@ def test_happy_switch(smartlight):
         (1, [45, 2]),
     ],
 )
-def test_interrogate_fiber(smartlight, inputportport, outputport):
+def test_interrogate_fiber(allegro, inputportport, outputport):
     with pytest.raises(AllegroConnectionError):
-        smartlight.interrogate_fiber(inputportport, outputport)
-    smartlight.connect()
+        allegro.interrogate_fiber(inputportport, outputport)
+    allegro.connect()
     with pytest.raises(AllegroError):
-        smartlight.interrogate_fiber(inputportport, outputport)
+        allegro.interrogate_fiber(inputportport, outputport)
 
 
-def test_happy_interrogate_fiber_1(smartlight):
-    smartlight.connect()
-    a = smartlight.interrogate_fiber(1, [1, 10])
+def test_happy_interrogate_fiber_1(allegro):
+    allegro.connect()
+    a = allegro.interrogate_fiber(1, [1, 10])
     assert isinstance(a, dict)
     for k, v in a.items():
         assert isinstance(k, int)
         assert isinstance(v, float)
 
 
-def test_happy_interrogate_fiber_2(smartlight):
-    smartlight.connect()
-    a = smartlight.interrogate_fiber(1)
+def test_happy_interrogate_fiber_2(allegro):
+    allegro.connect()
+    a = allegro.interrogate_fiber(1)
     assert isinstance(a, dict)
     for k, v in a.items():
         assert isinstance(k, int)
         assert isinstance(v, float)
+
+
+def test_get_temp(allegro):
+    with pytest.raises(AllegroConnectionError):
+        allegro.get_temp()
+    allegro.connect()
+    t = allegro.get_temp()
+    assert 35 <= t <= 50
+
+
+def test_output_power(allegro):
+    with pytest.raises(AllegroConnectionError):
+        allegro.get_output_power([1, 2, 3])
+    allegro.connect()
+    a = allegro.get_output_power([1, 2, 3])
+    assert isinstance(a, dict)
+    assert set(a.keys()) & set(a.values())
```

Comment:
I think this test is not doing what you intend

Reply:
done